webassembly-jpeg.c

```diff
@@ -8,6 +8,8 @@ Image *pSrcImage;
 
 Image *EMSCRIPTEN_KEEPALIVE setSrcImage(BYTE *jpegData, ULONG size)
 {
+    free(pSrcImage->data);
```

Comment:
pSrcImage should be init to NULL if the free should be called only if it is not null